db/migrate/20160303155656_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration
+  def change
+    create_table :users do |t|
+      t.string :username null: false 
+      t.string :password null: false
+      t.string :email null: false
+      t.integer :upVote nullL:false default:0
+      t.integer :downVote null:false default:1
```

Comment:
I suspect it will be easier for now just to do a count on their upvotes. Or the upvotes of their contributions.